dm/config/subtask.go

```diff
@@ -333,6 +334,13 @@ func (c *SubTaskConfig) Adjust(verifyDecryptPassword bool) error {
 			return terror.ErrConfigLoaderDirInvalid.Delegate(err, c.LoaderConfig.Dir)
 		}
 		c.LoaderConfig.Dir = newDir
+
+		if storage.IsLocalDiskPath(newDir) {
+			// lightning will not recursively create directories, so we use same level dir
+			c.LoaderConfig.SortingDirPhysical = newDir + ".sorting"
```

Comment:
Is `newDir` the data source directory? Does it affect the mydump read process?

Reply:
yes. `./dump_data` and `./dump_data.sorting` are two directories, it should not be affected